README.md

```diff
@@ -147,6 +147,14 @@ A curated list of awesome Java frameworks, libraries and software.
 * [Cobertura](http://cobertura.github.io/cobertura/) - Relies on offline (or static) bytecode instrumentation and class loading to collect code coverage metrics; GPLv2 licensed.
 * [JaCoCo](http://eclemma.org/jacoco/) - Framework that enables collection of code coverage metrics, using both offline and runtime bytecode instrumentation; prominently used by EclEmma, the Eclipse code-coverage plugin.
 
+## Code Generators
+
+*Code that generates bug-prone boilerplate code.*
```

Comment:
Didn't mean to be harsh or opinionated! For context, that's just my understanding of code generation: If code is amenable to being generated, then it's probably the kind of code that is repetitive and therefore bug-prone.

But, that's just my take. Not looking to step on toes for that perspective. Please change it to whatever you want or let me know what it should say, if anything.